src/widgets/prowlarr/ProwlarrTile.tsx

```diff
@@ -0,0 +1,87 @@
+import { Box, Button, Card, Flex, Group, Text } from '@mantine/core';
+import { IconCircleCheck, IconCircleX, IconReportSearch, IconTestPipe } from '@tabler/icons-react';
+import { useTranslation } from 'next-i18next';
+import { useConfigContext } from '~/config/provider';
+import { api } from '~/utils/api';
+
+import { defineWidget } from '../helper';
+import { WidgetLoading } from '../loading';
+import { IWidget } from '../widgets';
+
+const definition = defineWidget({
+  id: 'prowlarr',
+  icon: IconReportSearch,
+  options: {},
+  gridstack: {
+    minWidth: 1,
+    minHeight: 1,
+    maxWidth: 2,
```

Comment:
Use bigger size? I don't think that there is any advantage in limiting it to this small size